python/src/IterativeThresholdExceedance_doc.i.in

```diff
@@ -0,0 +1,70 @@
+%feature("docstring") OT::IterativeThresholdExceedance
+"Iterative minimum and maximum.
+
+
+Parameters
+----------
+dimension : int, default=1
+    Dimension of the input data
+
+thresholdValue : float, default=0.0
+    Threshold value
+
+Notes
+-----
+This class iteratively counts the threshold exceedance of the input :class:`~openturns.Point` or :class:`~openturns.Sample`.
+
+This class does not store the input data in memory.
+
+See also
+--------
+IterativeAlgorithm
+
+Examples
+--------
+>>> import openturns as ot
+>>> ot.RandomGenerator.SetSeed(0)
+>>> dim = 5
+>>> threshold =0.5
+>>> myThresholdExceedance = ot.IterativeThresholdExceedance(dim, threshold)
+>>> n = ot.Normal(dim)
+>>> size = 50
+
+Increment with 50 points:
+>>> for i in range(size):
+...     point = n.getRealization()
+...     myThresholdExceedance.increment(point)
+
```

Comment:
The examples are badly formatted: check the HTML output.

Reply:
Done